lib/tasks/task_master.rake

```diff
@@ -7,6 +8,17 @@ namespace :task_master do
     end
   end
 
+  task count_all: :environment do
+    Account.all.each do |account|
```

Comment:
Do we also want to take into account @mankind Suggestion of skipping search tenants 
```suggestion
    Account.where(search_only: false).each do |account|
```